packages/dart/sshnoports/bundles/shell/systemd/sshnpd.service

```diff
@@ -16,36 +10,7 @@ Type=simple
 Restart=always
 RestartSec=3
 
-# Configuration of sshnpd service
-# This unit script is a template for the sshnpd background service.
-# You can configure the service by editing the variables below.
-# This service file covers the common configuration options for sshnpd.
-# To see all available options, run `sshnpd` with no arguments.
-
-# MANDATORY: User to run the daemon as
-User=<username>
-
-# MANDATORY: Manager/client address (atSign)
-Environment=manager_atsign="@example_client"
-
-# MANDATORY: Device address (atSign)
-Environment=device_atsign="@example_device"
-
-# OPTIONAL: Delegated access policy management
-Environment=delegate_policy=""
-
-# Device name
-Environment=device_name="default"
-
-# Comment if you don't want the daemon to update authorized_keys to include public keys sent by authorized manager atSigns
-Environment=s="-s"
-
-# Comment if you don't want the daemon to share various information with the manager atsign - e.g. username, version, etc - without the manager atSign needing to know this daemon's device name
-Environment=u="-u"
-
-# Comment to disable verbose logging
-Environment=v="-v"
-
-# The line below runs the sshnpd service, with the options set above.
-# You can edit this line to further customize the service to your needs.
-ExecStart=/usr/local/bin/sshnpd -a "$device_atsign" -m "$manager_atsign" -d "$device_name" "$delegate_policy" "$s" "$u" "$v"
+# The line below runs the sshnpd service, with the options set in
+# /etc/systemd/system/sshnpd.d/override.conf.
+# You can edit that config with: sudo systemctl edit sshnpd
+ExecStart=/usr/local/bin/sshnpd -a "$device_atsign" -m "$manager_atsign" -d "$device_name" "$delegate_policy" "$s" "$u" "$v" "$additional_args"
```

Comment:
One thing I've been meaning to address, which may be suitable as part of this changeset is that when we use policy only the manager configuration will be `-m ""` which prevents the daemon from starting.

Reply:
Maybe we can change it to: `$( [ -n "$manager_atsign" ] && "-m $manager_atsign" || "" )`, and have a similar wrapper for policy.

Reply:
This can also be done in a later PR

Reply:
I was thinking this should actually be something like `${manager_atsign:+-m "$manager_atsign"}`, but it turns out that [systemd doesn't support parameter expansion](https://unix.stackexchange.com/questions/216780/why-does-bash-parameter-expansion-not-work-inside-systemd-service-files)

One option might be to wrap our command line inside a shell invocation, but I'm wary of creating a deeper process tree.

Reply:
Seems like the safest option is to allow sshnpd to try to pull those from the environment if they aren't explicitly passed.